### PR TITLE
EOS-22560: Failed to restart rsyslog.service during post_install stage

### PR DESCRIPTION
**Pull request by mariyappanp**
## Problem Statement:
Failed to restart rsyslog.service during SSPL deployment - post_install stage
This is observed as an intermittent issue. When we rerun the same command the issue wont occur again.

## Solution Design:

Wait and retry restarting rsyslog.service

For major feature, post the link to the solution page on the confluence Monitor team's space  -->

## Coding:

* [x] Did you follow coding standard and verified it with flake8? Yes

* [x] Did your address all Codacy issues? Yes

## Testing:

* [x] Unittest updated and all unittests are passing?

* [x] Sanity/Self tests are updated (if applicable)?

* [x] Manual testing done covering happy path and non-happy path?

## Integration:

* [x] If there is any interface change, did you communicate it to other components?
* [x] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [x] Did you complete deployment test if applicable?

## PR checklist:
* [x] Did you add Jira number to the PR?

* [x] DCO and cla-signed?

### CODE REVIEW

**Review by shriya-deshmukh — APPROVED** [review]

**Review by satish-seagate — APPROVED** [review]

**Review by jjxsg — APPROVED** [review]

**Review by sandeepsgt — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/setup/sspl_post_install.py [hunk 1]
```diff
@@ -321,7 +323,19 @@ def configure_sspl_syslog(self):
...
-        self.dbus_service.restart('rsyslog.service')
+        attempt = 0
+        while attempt < 3:
+            attempt += 1
+            try:
+                self.dbus_service.restart('rsyslog.service')
+                break
+            except ServiceError as err:
+                if not attempt < 3:
+                    logger.critical("Restarting rsyslog.service is failed " \
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/setup/sspl_post_install.py:334]: no 'is' required

File: low-level/files/opt/seagate/sspl/setup/sspl_post_install.py [hunk 2]
```diff
@@ -321,7 +323,19 @@ def configure_sspl_syslog(self):
...
+                        "due to error, %s" % err)
+                    raise
+                logger.debug("Waiting for rsyslog service to become active..")
+                time.sleep(2)
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/setup/sspl_post_install.py:338]: good, this sleep is imp, can make a difference based on observations till now

File: low-level/files/opt/seagate/sspl/setup/sspl_post_install.py (hunk 1 above)
Comment (mariyappanp) [reply low-level/files/opt/seagate/sspl/setup/sspl_post_install.py]: Done

**Review by cdeshmukh — APPROVED** [review]

**Comment by cdeshmukh** [comment]
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Cortx-Deployment/job/SSPL/429/
vm deployment passed

**Pull request merged**